packages/core/src/routes/sso-connector/index.ts

```diff
@@ -52,34 +51,21 @@ export default function singleSignOnRoutes<T extends AuthedRouter>(...args: Rout
   const pathname = `/${tableToPathname(SsoConnectors.table)}`;
 
   /*
-    Get all supported single sign on connector factory details
+    Get all supported single sign on connector provider details
     - standardConnectors: OIDC, SAML, etc.
     - providerConnectors: Google, Okta, etc.
   */
   router.get(
-    '/sso-connector-factories',
+    '/sso-connector-providers',
     koaGuard({
-      response: ssoConnectorFactoriesResponseGuard,
+      response: ssoConnectorProvidersResponseGuard,
       status: [200],
     }),
     async (ctx, next) => {
       const { locale } = ctx;
       const factories = Object.values(ssoConnectorFactories);
```

Comment:
Should we also rename these `factories` to `providers`?

Reply:
Internal code reference should be fine. Looks straight forward. Just need to hide the factories from the API/data level. factories more like our internal connector logic.